superset/assets/src/SqlLab/components/AceEditorWrapper.jsx

```diff
@@ -27,6 +27,11 @@ import { areArraysShallowEqual } from '../../reduxUtils';
 
 const langTools = ace.acequire('ace/ext/language_tools');
 
+const SQL_KEYWORD_AUTOCOMPLETE_SCORE = 100;
+const SCHEMA_AUTOCOMPLETE_SCORE = 60;
+const TABLE_AUTOCOMPLETE_SCORE = 55;
+const COLUMN_AUTOCOMPLETE_SCORE = 50;
```

Comment:
When writing a query, one is probably spending more time autocompleting column names than table/schema names. Wondering if having a higher score for column names wrt schema/table names could result in a better user experience, too?

Reply:
Hmm, I'm not sure about this. I'm going to merge this refactor in, reach out to some sql lab powerusers, and come back with the results. I anticipate making some more changes here as we have other feedback about autocomplete to fix.

Reply:
Absolutely; this was not meant as a blocking comment. Btw, I briefly checked DataGrip, and it seems it gives a higher score/priority for columns than tables, so I might be on to something here after all 😄